app/models/application_notifier.rb

```diff
@@ -31,10 +33,7 @@ def insert_log_url
 
   # use the same redis instance we use for caching
   def self.redis
-    Redis.new Rails.application.config_for(:cache_store).merge(
-      timeout: 1,
-      ssl: (ENV.fetch('CACHE_SSL') { 'false' }) == 'true',
-    )
+    Rails.cache.redis
```

Comment:
Use Rails.cache.redis; this is now a connection pool so in theory it should more reliable